flutter/lib/src/sentry_screenshot_options.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:async';
+
+import '../sentry_flutter.dart';
+
+/// Configuration of the screenshot feature.
+class SentryScreenshotOptions {
+  /// Automatically attaches a screenshot when capturing an error or exception.
+  ///
+  /// Requires adding the [SentryScreenshotWidget] to the widget tree.
+  /// Example:
+  /// runApp(SentryScreenshotWidget(child: App()));
+  /// The [SentryScreenshotWidget] has to be the root widget of the app.
```

Comment:
We should point users to use the single root widget:
```suggestion
  /// The [SentryWidget] has to be the root widget of the app.
```